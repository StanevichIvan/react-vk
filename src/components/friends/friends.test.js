import renderer from 'react-test-renderer';
import Friends from './friends';
import * as React from "react";

describe('Friends rendering' , () => {

    it('renders correctly', () => {
        const rendered = renderer.create(<Friends/>);
        expect(rendered.toJSON()).toMatchSnapshot();
    });
});
