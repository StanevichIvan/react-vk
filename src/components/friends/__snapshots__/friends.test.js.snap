exports[`Friends rendering renders correctly 1`] = `
<div
  className="friends-list-container">
  <form
    onSubmit={[Function]}>
    <input
      type="text" />
    <input
      name="search"
      onClick={[Function]}
      type="button"
      value="Search" />
  </form>
</div>
`;
